eslint.config.js

```diff
@@ -1,25 +1,32 @@
 const ts = require('typescript-eslint');
 const js = require('@eslint/js');
 const prettier = require('eslint-config-prettier');
-const cypress = require('eslint-plugin-cypress');
+const cypress = require('eslint-plugin-cypress/flat');
+const jest = require('eslint-plugin-jest');
 
 module.exports = ts.config(
   js.configs.recommended,
   ...ts.configs.recommended,
   prettier,
-  {
-    plugins: { cypress: cypress },
-  },
   {
     ignores: ['lib/**'],
   },
   {
-    files: ['**/*.js', '**/*.ts'],
+    files: ['**/*.*js', '**/*.*ts'],
     rules: {
       '@typescript-eslint/no-var-requires': 'off',
+      '@typescript-eslint/no-explicit-any': 'warn',
```

Comment:
We currently don't have any explicit `any` and the default rule is to error. You are relaxing this rule to `warn`, but why though? We have some legacy projects that transitioned from js to ts, and to allow for a more gradual transformation, we relaxed the rules there so that we can slowly move away from the use of `any`, not to allow it indefinitely.